src/components/Display.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = props => {
+  const { result } = props;
+  return (
+    <>
+      <div className="display">
+        {result}
+      </div>
+    </>
```

Comment:
- [ ] React Fragment it's not needed here since we are returning only one parent JSX element: `div`